app/graphql/mocks/customer.js

```diff
@@ -26,8 +26,15 @@ export const Customer = () => {
   }
 }
 
-export const CustomerInfo = () => {
-  return {
-    dateOfBirth: faker.string.numeric()
-  }
-}
+export const CustomerInfo = () => ({
```

Comment:
this is all removed in my next pr so can you remove your changes to avoide conflicts?